Readme.md

```diff
@@ -49,11 +49,69 @@ Similarly, you can execute `npm run dotnet-postclean --if-present` via the `dotn
 ```
 
 
-## Using pnpm for package restore
+## Using PNPM for package restore
 
-[Pnpm](https://pnpm.io/) is a faster and more efficient package manager. If it's installed globally, then the module will use that instead of *npm* to restore packages.
+[PNPM](https://pnpm.io/) is a faster and more efficient package manager. If it's installed globally, then the module will use `pnpm` instead of `npm` to restore packages.
 
-To install *pnpm* globally run this command: `npm install pnpm -g`. Once it's complete, the module will automatically use that to restore packages.
+To install `pnpm` globally run this command: `npm install pnpm -g`. Once it's complete, the module will automatically use that to restore packages.
```

Comment:
Here, it's the tool name in contrast to the command name:
```suggestion
To install PNPM globally run this command: `npm install pnpm -g`. Once it's complete, the module will automatically use that to restore packages.
```

Reply:
Why did you not update this instance?

Reply:
I just noticed, the same thing is present in two different files:
![image](https://user-images.githubusercontent.com/4007293/170365339-b96b2192-9e97-4f49-9f32-b02d82feaf34.png)

Presumably I have updated the wrong one. Let me think on which one needs to be removed.

Reply:
Should be good now. Gulp Extensions had more generic PNPM information which I moved here as well.